src/builder/create_interaction_response_followup.rs

```diff
@@ -90,15 +90,21 @@ impl<'a> CreateInteractionResponseFollowup<'a> {
     {
         let mut embed = CreateEmbed::default();
         f(&mut embed);
-        self.set_embed(embed)
+        self.add_embed(embed)
     }
 
-    /// Set an embed for the message.
-    pub fn set_embed(&mut self, embed: CreateEmbed) -> &mut Self {
+    /// Adds an embed to the message.
+    pub fn add_embed(&mut self, embed: CreateEmbed) -> &mut Self {
```

Comment:
You should also create a separate `set_embeds` method which would take an array of embeds.

Reply:
Created an `embeds` method to go with the `files` method.